diffkemp/building/cc_wrapper.py

```diff
@@ -9,10 +9,10 @@
 """
 # TODO: Decompose simultaneous compiling and linking to get individual .ll
 #       files out of it; support other GCC languages than C (Fortran, C++)
-from diffkemp.llvm_ir.compiler import get_clang_default_options
 import os
 import shutil
 import sys
+import diffkemp
```

Comment:
This will probably be a problem when the file is compiled with RPython because IIRC RPython needs all imported files to have Python 2 compatible syntax.